bower.json

```diff
@@ -10,32 +10,5 @@
   "repository": {
```

Comment:
Is there a reason not to remove the bower.json file altogether?

Reply:
I would prefer if we keep the bower files for now, unless we're going to make all these updates breaking releases too, as I think it'll break existing projects that use bower otherwise. Like my main work project. :smile:

Reply:
@thomashoneyman

> I don't see why we would keep a bowerfile but remove all the dependencies from it

some bowerfiles contained the `authors: [....]` field, and it is not available for spago.yaml, so, just wanted to show

maybe add https://github.com/purescript/spago/issues/1294?

@garyb 

> I think it'll break existing projects that use bower otherwise

but I am in a process of updating the all contrib libraries

btw, my scripts I use

https://github.com/srghma/spago-yaml-generate/blob/main/bin/bower-json-to-spago-yaml

https://github.com/srghma/dotfiles/blob/main/bin/clone-many.sh

Reply:
> I would prefer if we keep the bower files for now, unless we're going to make all these updates breaking releases too, as I think it'll break existing projects that use bower otherwise. Like my main work project.

I agree with making these breaking changes in any case.

How would you feel about removing the bowerfiles, marking this as a breaking change, and cutting a "stable" release of the new spago?

Reply:
can move the "authors" field from bower.json to then end of README.md and remove bower.json from all projects